rpc/http.go

```diff
@@ -37,6 +37,7 @@ type HTTPServerConfig struct {
 	StorageAPI modules.StorageAPI
 	NetworkAPI modules.NetworkAPI
 	CoreAPI    modules.CoreAPI
+	TxQueueAPI modules.TransactionQueueAPI
```

Comment:
Again, I would probably write out the full `TransactionQueueAPI` given its a configuration struct.

Reply:
updated